core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/Server.java

```diff
@@ -25,25 +25,35 @@
     String getRef();
 
     /**
-     * Address of the server in form <b>host:port</b>
+     * External address of the server in form <b>host:port</b>
+     * This address is used by the browser to communicate with the server and
+     * corresponds to the internal address if not explicitly set.
+     * Can be set using property machine.docker.local_node_host.external or
+     * environment variable CHE_DOCKER_MACHINE_HOST_EXTERNAL
      */
     String getAddress();
```

Comment:
Now this method returns both host and port of the server. Are you going to change the behavior in this PR?